src/main/java/it/pagopa/interop/probing/eservice/operations/util/EnumUtilities.java

```
@@ -19,14 +19,11 @@ public class EnumUtilities {
   private int minOfTolleranceMultiplier;
 
   public static String fromMonitorToPdndState(EserviceMonitorState state) {
-    switch (state) {
-      case ONLINE:
-        return EserviceInteropState.ACTIVE.getValue();
-      case OFFLINE:
-        return EserviceInteropState.INACTIVE.getValue();
-      default:
-        return null;
-    }
+    return switch (state) {
+      case ONLINE -> EserviceInteropState.ACTIVE.getValue();
+      case OFFLINE -> EserviceInteropState.INACTIVE.getValue();
+      default -> null;
```

Maybe it is better to throw an exception, null is not an option.